markets/perps-market/test/integration/helpers/funding-calcs.ts

```diff
@@ -0,0 +1,22 @@
+import Wei, { wei } from '@synthetixio/wei';
+
+export function calcCurrentFundingVelocity({
+  skew,
+  skewScale,
+  maxFundingVelocity,
+}: {
+  skew: Wei;
+  skewScale: Wei;
+  maxFundingVelocity: Wei;
+}) {
+  // Avoid a panic due to div by zero. Return 0 immediately.
+  if (skewScale.eq(0)) {
+    return wei(0);
+  }
+
+  // Ensures the proportionalSkew is between -1 and 1.
+  const pSkew = wei(skew).div(skewScale);
+  const pSkewBounded = Wei.max(Wei.min(pSkew, wei(1)), wei(-1));
+
+  return pSkewBounded.mul(maxFundingVelocity);
```

Comment:
maybe returns bignumber via `.toBN()` here?

Reply:
Coming from the frontend we usually keep everything in Wei and just `toBN` before sending to ethers contract call. Wei usually makes math and most things easier. 
But we do seem to like/use BigNumber more in contract tests, so I guess it make sense to return bn.

Reply:
ok cool yeah fair enough!